packages/scandipwa/src/route/UrlRewrites/UrlRewrites.component.js

```diff
@@ -81,33 +72,83 @@ export class UrlRewrites extends PureComponent {
                   productID={ id }
                   key={ id }
                 />
-            );
+            ) : this.renderDefaultPage()
```

Comment:
Please follow this pattern when you implement the render method with conditional components.
```javascript
renderSomething() {
  const { someCondition } = this.props;

  if (!someCondition) {
    return <RenderSomethingDefault />;
  }

  return <RenderSomethingUsefull>;
}
```